assets/sass/scaffolding/grid.scss

```diff
@@ -224,3 +224,40 @@ $num_columns_mobile: 3;
         }
     }
 }
+
+/* =========================================================================
+   Flex Grid
+   ========================================================================= */
+
+.flex-grid {
```

Comment:
Is this similar to `.grid {}`?

Reply:
Hmm…yeah not sure why I thought it was useful to have a different one. Let me take another look.

Reply:
Looking at this again, the reason I went down this route is because the current grid has some issues with grid items that take up an equal height. This is probably fixable in general terms, but the number of generated classes and modifiers of the current grid system gives me pause as it feels a bit too magic each time I  use it. I think this is mostly a documentation thing, but for now I'd prefer to use something simpler here and coming back to do some work to make sense of the existing grid styles.